### ISSUE TITLE
Can't find crate for "cheddar"

### ISSUE THREAD

**Issue by HarvHouHacker** [HarvHouHacker/Light-Web-Browser#10]
Got this error when attempting `./mach build`:
```
10:34.05 error[E0463]: can't find crate for `cheddar`
10:34.05  --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs:1:1
10:34.05   |
10:34.05 1 | extern crate cheddar;
10:34.05   | ^^^^^^^^^^^^^^^^^^^^^ can't find crate
10:34.05 
10:34.05 error: aborting due to previous error
10:34.05 
10:34.09 error: Could not compile `mp4parse_capi`.
10:34.09 
10:34.09 Caused by:
10:34.09   process didn't exit successfully: `$HOME_DIR$/.cargo/bin/rustc --crate-name build_script_build $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs --color always --crate-type bin --emit=dep-info,link -C opt-level=2 -C codegen-units=1 -C debuginfo=2 -C metadata=ba641d6176b1a272 -C extra-filename=-ba641d6176b1a272 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/build/mp4parse_capi-ba641d6176b1a272 -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps` (exit code: 101)
10:34.10 Build failed, waiting for other jobs to finish...
10:36.86 error: build failed
10:36.86 $HOME_DIR$/firefox-53.0.3/config/rules.mk:966: recipe for target 'force-cargo-library-build' failed
10:36.86 make[5]: *** [force-cargo-library-build] Error 101
10:36.86 $HOME_DIR$/firefox-53.0.3/config/recurse.mk:71: recipe for target 'toolkit/library/gtest/rust/target' failed
10:36.86 make[4]: *** [toolkit/library/gtest/rust/target] Error 2
10:36.86 make[4]: *** Waiting for unfinished jobs....
10:48.03 libdom_presentation.a.desc
10:48.22 $HOME_DIR$/firefox-53.0.3/config/recurse.mk:32: recipe for target 'compile' failed
10:48.22 make[3]: *** [compile] Error 2
10:48.22 $HOME_DIR$/firefox-53.0.3/config/rules.mk:522: recipe for target 'default' failed
10:48.23 make[2]: *** [default] Error 2
10:48.23 $HOME_DIR$/firefox-53.0.3/client.mk:419: recipe for target 'realbuild' failed
10:48.23 make[1]: *** [realbuild] Error 2
10:48.23 client.mk:170: recipe for target 'build' failed
10:48.23 make: *** [build] Error 2
10:48.23 0 compiler warnings present.
10:48.37 /usr/bin/notify-send --app-name=Mozilla Build System Mozilla Build System Build failed
```
Gonna have to resolve this somehow...

**Comment by HarvHouHacker** [comment]
This is an apparently a bug in older versions of Rust. I didn't have rustc installed, so I installed it via Ubuntu Software, deleted the .cargo directory in my home folder, and ran `rustc update`. I'll try again...

**Comment by HarvHouHacker** [comment]
Deja vu:
```
 8:43.82 error[E0463]: can't find crate for `cheddar`
 8:43.82  --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs:1:1
 8:43.82   |
 8:43.82 1 | extern crate cheddar;
 8:43.82   | ^^^^^^^^^^^^^^^^^^^^^ can't find crate
 8:43.82 
 8:43.82 error: aborting due to previous error
 8:43.82 
 8:43.86 error: Could not compile `mp4parse_capi`.
 8:43.86 
 8:43.86 Caused by:
 8:43.86   process didn't exit successfully: `$HOME_DIR$/.cargo/bin/rustc --crate-name build_script_build $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs --color always --crate-type bin --emit=dep-info,link -C opt-level=2 -C codegen-units=1 -C debuginfo=2 -C metadata=ba641d6176b1a272 -C extra-filename=-ba641d6176b1a272 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/build/mp4parse_capi-ba641d6176b1a272 -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps` (exit code: 101)
 8:43.86 Build failed, waiting for other jobs to finish...
 8:45.31 error: build failed
 8:45.31 $HOME_DIR$/firefox-53.0.3/config/rules.mk:966: recipe for target 'force-cargo-library-build' failed
 8:45.31 make[5]: *** [force-cargo-library-build] Error 101
 8:45.31 $HOME_DIR$/firefox-53.0.3/config/recurse.mk:71: recipe for target 'toolkit/library/gtest/rust/target' failed
 8:45.31 make[4]: *** [toolkit/library/gtest/rust/target] Error 2
 8:45.31 make[4]: *** Waiting for unfinished jobs....
 8:57.16 libdom_presentation.a.desc
 8:58.23 $HOME_DIR$/firefox-53.0.3/config/recurse.mk:32: recipe for target 'compile' failed
 8:58.23 make[3]: *** [compile] Error 2
 8:58.23 $HOME_DIR$/firefox-53.0.3/config/rules.mk:522: recipe for target 'default' failed
 8:58.23 make[2]: *** [default] Error 2
 8:58.33 $HOME_DIR$/firefox-53.0.3/client.mk:419: recipe for target 'realbuild' failed
 8:58.33 make[1]: *** [realbuild] Error 2
 8:58.33 client.mk:170: recipe for target 'build' failed
 8:58.33 make: *** [build] Error 2
 8:58.34 0 compiler warnings present.
 8:58.49 /usr/bin/notify-send --app-name=Mozilla Build System Mozilla Build System Build failed
```
Now I understand; it's a problem with`cargo`: https://reviewboard.mozilla.org/r/111682/#review113446 Looks like I may have to downgrade. 😞

**Comment by HarvHouHacker** [comment]
Apparently, Rust is built-in to the source code, so regardless of my system-wide `rustc` and `cargo`, it will always use what is in source. That stinks! It doesn't work; I ran a test:
```
$ ./mach gtest rust.*
 0:00.64 /usr/bin/make -C testing/gtest -j2 -s gtest
 0:02.59 force-cargo-library-build
 0:02.96 BUILDSTATUS OBJECT_FILE file_id.o
 0:03.66    Compiling byteorder v1.0.0
 0:03.66        Fresh bitreader v0.2.0
 0:03.66    Compiling matches v0.1.3
 0:03.66      Running `/usr/local/bin/rustc --crate-name byteorder $HOME_DIR$/firefox-53.0.3/third_party/rust/byteorder/src/lib.rs --color always --crate-type lib --emit=dep-info,link -C opt-level=2 -C panic=abort -C codegen-units=1 -C debuginfo=2 --cfg 'feature="default"' --cfg 'feature="std"' -C metadata=95d3a46fdaf18ed4 -C extra-filename=-95d3a46fdaf18ed4 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps --target x86_64-unknown-linux-gnu -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps --cap-lints allow`
 0:03.67      Running `/usr/local/bin/rustc --crate-name matches $HOME_DIR$/firefox-53.0.3/third_party/rust/matches/lib.rs --color always --crate-type lib --emit=dep-info,link -C opt-level=2 -C panic=abort -C codegen-units=1 -C debuginfo=2 -C metadata=855ef2506f9928fb -C extra-filename=-855ef2506f9928fb --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps --target x86_64-unknown-linux-gnu -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps --cap-lints allow`
 0:04.18    Compiling unicode-bidi v0.2.3
 0:04.18      Running `/usr/local/bin/rustc --crate-name unicode_bidi $HOME_DIR$/firefox-53.0.3/third_party/rust/unicode-bidi/src/lib.rs --color always --crate-type lib --emit=dep-info,link -C opt-level=2 -C panic=abort -C codegen-units=1 -C debuginfo=2 -C metadata=80e81e0e897e9151 -C extra-filename=-80e81e0e897e9151 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps --target x86_64-unknown-linux-gnu -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps --extern matches=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps/libmatches-855ef2506f9928fb.rlib --cap-lints allow`
 0:04.62    Compiling mp4parse_capi v0.6.0 (file://$HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi)
 0:04.62      Running `/usr/local/bin/rustc --crate-name build_script_build $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs --color always --crate-type bin --emit=dep-info,link -C opt-level=2 -C codegen-units=1 -C debuginfo=2 -C metadata=decef34fa7f8c368 -C extra-filename=-decef34fa7f8c368 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/build/mp4parse_capi-decef34fa7f8c368 -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps`
 0:04.77 error[E0463]: can't find crate for `cheddar`
 0:04.77  --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs:1:1
 0:04.77   |
 0:04.77 1 | extern crate cheddar;
 0:04.77   | ^^^^^^^^^^^^^^^^^^^^^ can't find crate
 0:04.77 
 0:04.77 error: aborting due to previous error
 0:04.77 
 0:04.78 error: Could not compile `mp4parse_capi`.
 0:04.78 
 0:04.78 Caused by:
 0:04.78   process didn't exit successfully: `/usr/local/bin/rustc --crate-name build_script_build $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/build.rs --color always --crate-type bin --emit=dep-info,link -C opt-level=2 -C codegen-units=1 -C debuginfo=2 -C metadata=decef34fa7f8c368 -C extra-filename=-decef34fa7f8c368 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/build/mp4parse_capi-decef34fa7f8c368 -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps` (exit code: 101)
 0:04.78 Build failed, waiting for other jobs to finish...
 0:04.98 BUILDSTATUS OBJECT_FILE Unified_cpp_src_common_linux0.o
 0:05.85 libbreakpad_linux_common_s.a.desc
 0:06.16 BUILDSTATUS OBJECT_FILE Unified_cpp_components_places0.o
 0:08.38 error: build failed
 0:08.39 $HOME_DIR$/firefox-53.0.3/config/rules.mk:966: recipe for target 'force-cargo-library-build' failed
 0:08.39 make[3]: *** [force-cargo-library-build] Error 101
 0:08.39 $HOME_DIR$/firefox-53.0.3/config/recurse.mk:71: recipe for target 'toolkit/library/gtest/rust/target' failed
 0:08.39 make[2]: *** [toolkit/library/gtest/rust/target] Error 2
 0:08.39 make[2]: *** Waiting for unfinished jobs....
 0:30.85 libtoolkit_components_places.a.desc
 0:31.44 Makefile:25: recipe for target 'gtestxul' failed
 0:31.44 make[1]: *** [gtestxul] Error 2
 0:31.44 Makefile:21: recipe for target 'gtest' failed
 0:31.44 make: *** [gtest] Error 2
Error running mach:

    ['gtest', 'rust.*']

The error occurred in code that was called by the mach command. This is either
a bug in the called code itself or in the way that mach is calling it.

You should consider filing a bug for this issue.

If filing a bug, please include the full output of mach, including this error
message.

The details of the failure are as follows:

Exception: Process executed with non-0 exit code 2: [u'/usr/bin/make', u'-C', u'testing/gtest', u'-j2', u'-s', u'gtest']

  File "$HOME_DIR$/firefox-53.0.3/python/mozbuild/mozbuild/mach_commands.py", line 898, in gtest
    print_directory=False, ensure_exit_code=True)
  File "$HOME_DIR$/firefox-53.0.3/python/mozbuild/mozbuild/base.py", line 548, in _run_make
    return fn(**params)
  File "$HOME_DIR$/firefox-53.0.3/python/mozbuild/mozbuild/base.py", line 603, in _run_command_in_objdir
    return self.run_process(cwd=self.topobjdir, **args)
  File "$HOME_DIR$/firefox-53.0.3/python/mach/mach/mixin/process.py", line 147, in run_process
    raise Exception('Process executed with non-0 exit code %d: %s' % (status, args))
```

**Comment by HarvHouHacker** [comment]
Didn't really want to, but I finally patched the Cargo.toml files manually. Result:
```
 0:00.95    Compiling mp4parse_capi v0.6.0 (file://$HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi)
 0:00.97      Running `/usr/local/bin/rustc --crate-name mp4parse_capi $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs --color always --crate-type lib --emit=dep-info,link -C opt-level=2 -C panic=abort -C codegen-units=1 -C debuginfo=2 -C metadata=519516728715f3d7 -C extra-filename=-519516728715f3d7 --out-dir $HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps --target x86_64-unknown-linux-gnu -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps -L dependency=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./release/deps --extern mp4parse=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps/libmp4parse-af0773c488dcb562.rlib --extern byteorder=$HOME_DIR$/firefox-53.0.3/obj-x86_64-pc-linux-gnu/toolkit/library/gtest/rust/./x86_64-unknown-linux-gnu/release/deps/libbyteorder-95d3a46fdaf18ed4.rlib`
 0:01.51 warning: variant `MP4PARSE_OK` should have a camel case name such as `Mp4parseOk`
 0:01.51   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:70:5
 0:01.51    |
 0:01.51 70 |     MP4PARSE_OK = 0,
 0:01.51    |     ^^^^^^^^^^^^^^^
 0:01.51    |
 0:01.51    = note: #[warn(non_camel_case_types)] on by default
 0:01.51 
 0:01.51 warning: variant `MP4PARSE_ERROR_BADARG` should have a camel case name such as `Mp4parseErrorBadarg`
 0:01.51   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:71:5
 0:01.51    |
 0:01.51 71 |     MP4PARSE_ERROR_BADARG = 1,
 0:01.51    |     ^^^^^^^^^^^^^^^^^^^^^^^^^
 0:01.51    |
 0:01.51    = note: #[warn(non_camel_case_types)] on by default
 0:01.51 
 0:01.51 warning: variant `MP4PARSE_ERROR_INVALID` should have a camel case name such as `Mp4parseErrorInvalid`
 0:01.51   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:72:5
 0:01.51    |
 0:01.51 72 |     MP4PARSE_ERROR_INVALID = 2,
 0:01.51    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^
 0:01.51    |
 0:01.51    = note: #[warn(non_camel_case_types)] on by default
 0:01.51 
 0:01.51 warning: variant `MP4PARSE_ERROR_UNSUPPORTED` should have a camel case name such as `Mp4parseErrorUnsupported`
 0:01.51   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:73:5
 0:01.51    |
 0:01.51 73 |     MP4PARSE_ERROR_UNSUPPORTED = 3,
 0:01.51    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 0:01.51    |
 0:01.51    = note: #[warn(non_camel_case_types)] on by default
 0:01.51 
 0:01.51 warning: variant `MP4PARSE_ERROR_EOF` should have a camel case name such as `Mp4parseErrorEof`
 0:01.51   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:74:5
 0:01.51    |
 0:01.51 74 |     MP4PARSE_ERROR_EOF = 4,
 0:01.51    |     ^^^^^^^^^^^^^^^^^^^^^^
 0:01.51    |
 0:01.51    = note: #[warn(non_camel_case_types)] on by default
 0:01.51 
 0:01.51 warning: variant `MP4PARSE_ERROR_IO` should have a camel case name such as `Mp4parseErrorIo`
 0:01.52   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:75:5
 0:01.52    |
 0:01.52 75 |     MP4PARSE_ERROR_IO = 5,
 0:01.52    |     ^^^^^^^^^^^^^^^^^^^^^
 0:01.52    |
 0:01.52    = note: #[warn(non_camel_case_types)] on by default
 0:01.52 
 0:01.52 warning: variant `MP4PARSE_TRACK_TYPE_VIDEO` should have a camel case name such as `Mp4parseTrackTypeVideo`
 0:01.52   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:81:5
 0:01.52    |
 0:01.52 81 |     MP4PARSE_TRACK_TYPE_VIDEO = 0,
 0:01.52    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 0:01.52    |
 0:01.53    = note: #[warn(non_camel_case_types)] on by default
 0:01.53 
 0:01.53 warning: variant `MP4PARSE_TRACK_TYPE_AUDIO` should have a camel case name such as `Mp4parseTrackTypeAudio`
 0:01.53   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:82:5
 0:01.53    |
 0:01.53 82 |     MP4PARSE_TRACK_TYPE_AUDIO = 1,
 0:01.53    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 0:01.53    |
 0:01.53    = note: #[warn(non_camel_case_types)] on by default
 0:01.53 
 0:01.53 warning: variant `MP4PARSE_CODEC_UNKNOWN` should have a camel case name such as `Mp4parseCodecUnknown`
 0:01.53   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:92:5
 0:01.53    |
 0:01.53 92 |     MP4PARSE_CODEC_UNKNOWN,
 0:01.53    |     ^^^^^^^^^^^^^^^^^^^^^^
 0:01.53    |
 0:01.53    = note: #[warn(non_camel_case_types)] on by default
 0:01.53 
 0:01.53 warning: variant `MP4PARSE_CODEC_AAC` should have a camel case name such as `Mp4parseCodecAac`
 0:01.53   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:93:5
 0:01.53    |
 0:01.53 93 |     MP4PARSE_CODEC_AAC,
 0:01.53    |     ^^^^^^^^^^^^^^^^^^
 0:01.53    |
 0:01.53    = note: #[warn(non_camel_case_types)] on by default
 0:01.53 
 0:01.54 warning: variant `MP4PARSE_CODEC_FLAC` should have a camel case name such as `Mp4parseCodecFlac`
 0:01.54   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:94:5
 0:01.54    |
 0:01.54 94 |     MP4PARSE_CODEC_FLAC,
 0:01.54    |     ^^^^^^^^^^^^^^^^^^^
 0:01.54    |
 0:01.54    = note: #[warn(non_camel_case_types)] on by default
 0:01.54 
 0:01.54 warning: variant `MP4PARSE_CODEC_OPUS` should have a camel case name such as `Mp4parseCodecOpus`
 0:01.54   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:95:5
 0:01.54    |
 0:01.54 95 |     MP4PARSE_CODEC_OPUS,
 0:01.54    |     ^^^^^^^^^^^^^^^^^^^
 0:01.54    |
 0:01.54    = note: #[warn(non_camel_case_types)] on by default
 0:01.54 
 0:01.54 warning: variant `MP4PARSE_CODEC_AVC` should have a camel case name such as `Mp4parseCodecAvc`
 0:01.54   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:96:5
 0:01.54    |
 0:01.54 96 |     MP4PARSE_CODEC_AVC,
 0:01.54    |     ^^^^^^^^^^^^^^^^^^
 0:01.55    |
 0:01.55    = note: #[warn(non_camel_case_types)] on by default
 0:01.55 
 0:01.55 warning: variant `MP4PARSE_CODEC_VP9` should have a camel case name such as `Mp4parseCodecVp9`
 0:01.55   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:97:5
 0:01.55    |
 0:01.55 97 |     MP4PARSE_CODEC_VP9,
 0:01.55    |     ^^^^^^^^^^^^^^^^^^
 0:01.55    |
 0:01.55    = note: #[warn(non_camel_case_types)] on by default
 0:01.55 
 0:01.55 warning: variant `MP4PARSE_CODEC_MP3` should have a camel case name such as `Mp4parseCodecMp3`
 0:01.55   --> $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/src/lib.rs:98:5
 0:01.55    |
 0:01.55 98 |     MP4PARSE_CODEC_MP3,
 0:01.55    |     ^^^^^^^^^^^^^^^^^^
 0:01.55    |
 0:01.55    = note: #[warn(non_camel_case_types)] on by default
```
Files I patched are in $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse/ and $HOME_DIR$/firefox-53.0.3/media/libstagefright/binding/mp4parse_capi/.
